src/frontend/src/catalog/pg_catalog/information_schema/columns.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_common::types::DataType;
```

Comment:
I think `information_schema` and `pg_catalog` should be on the same level, maybe under a folder named `system_catalog`?

Reply:
I have adjusted the directory structure.